Unable to run develop build on Windows platform

Trying to run the current version of develop on a Windows container platform will raise the following panic:

```
panic: Failed to load netapi32.dll: The specified module could not be found.
goroutine 1 [running]:
syscall.(*LazyProc).mustFind(0xc00006da10)
        C:/hostedtoolcache/windows/go/1.13.1/x64/src/syscall/dll_windows.go:311 +0x5f
syscall.(*LazyProc).Addr(0xc00006da10, 0x23)
        C:/hostedtoolcache/windows/go/1.13.1/x64/src/syscall/dll_windows.go:318 +0x32
syscall.NetGetJoinInformation(0x0, 0xc0001a1a30, 0xc0001a1a2c, 0x16, 0xc0001c13b0)
        C:/hostedtoolcache/windows/go/1.13.1/x64/src/syscall/zsyscall_windows.go:1804 +0x38
os/user.isDomainJoined(0xc0001a1a88, 0x4477de, 0x0)
        C:/hostedtoolcache/windows/go/1.13.1/x64/src/os/user/lookup_windows.go:18 +0x5a
os/user.lookupFullName(0xc000394a80, 0xc, 0xc000353020, 0x16, 0xc0001c13b0, 0x23, 0x16, 0xc0001c13b0, 0x23, 0x1)
        C:/hostedtoolcache/windows/go/1.13.1/x64/src/os/user/lookup_windows.go:55 +0x2d
os/user.newUser(0xc000394a50, 0xc, 0xc000394a70, 0xc, 0xc0001c1350, 0x1f, 0xc000353020, 0x16, 0xc000394a80, 0xc, ...)
        C:/hostedtoolcache/windows/go/1.13.1/x64/src/os/user/lookup_windows.go:186 +0xd4
os/user.current(0x0, 0x0, 0x0)
        C:/hostedtoolcache/windows/go/1.13.1/x64/src/os/user/lookup_windows.go:230 +0x2bd
os/user.Current.func1()
        C:/hostedtoolcache/windows/go/1.13.1/x64/src/os/user/lookup.go:15 +0x29
sync.(*Once).doSlow(0x2974060, 0x1acbe48)
        C:/hostedtoolcache/windows/go/1.13.1/x64/src/sync/once.go:66 +0xea
sync.(*Once).Do(...)
        C:/hostedtoolcache/windows/go/1.13.1/x64/src/sync/once.go:57
os/user.Current(0xc000394a40, 0xc, 0x19fd575)
        C:/hostedtoolcache/windows/go/1.13.1/x64/src/os/user/lookup.go:15 +0xf9
k8s.io/klog.init.1()
        d:/a/_temp/go/pkg/mod/k8s.io/klog@v1.0.0/klog_file.go:58 +0x48
```

It's probably related to the following topics:

* go issue: https://github.com/golang/go/issues/21867
* klog issue: https://github.com/kubernetes/klog/issues/122

This seems to have been fixed in klog v2.0.0:

* https://github.com/kubernetes/klog/pull/123
* https://github.com/kubernetes/klog/compare/v1.0.0...v2.0.0-rc.1

We're currently using `k8s.io/client-go v0.17.2` which embeds version 1.0.0 of klog: https://github.com/kubernetes/client-go/blob/bdedab45d4f6c8f8ffacb8e269c89e9bb736320f/go.mod

I believe this can be fixed by upgrading to a newer version of  `k8s.io/client-go`, I'm not able to properly identify which version of this library contains the new version of `klog` though... I think it's only available in v0.19.0+ but this version is still in release candidate state:

* https://github.com/kubernetes/client-go/commit/3d5c80942cce510064da1ab62c579e190a0230fd
* https://github.com/kubernetes/client-go/blob/v0.19.0-beta.0/go.mod
* https://github.com/kubernetes/client-go/blob/v0.18.5/go.mod

We could also work around this problem by copying the missing netapi32.dll into the nanoserver image:

* https://github.com/StefanScherer/dockerfiles-windows/pull/197

--- a/build/windows2016/nanoserver/Dockerfile
+++ b/build/windows2016/nanoserver/Dockerfile
@@ -1,4 +1,7 @@
+FROM mcr.microsoft.com/windows/servercore as core
 FROM microsoft/nanoserver:sac2016
+
+COPY --from=core /windows/system32/netapi32.dll
 
 COPY dist /
 

--- a/build/windows2016/nanoserver/Dockerfile
+++ b/build/windows2016/nanoserver/Dockerfile
@@ -1,6 +1,8 @@
 FROM mcr.microsoft.com/windows/servercore:ltsc2019 as core
 FROM mcr.microsoft.com/windows/nanoserver:1809-amd64
 
+USER ContainerAdministrator
+
 COPY --from=core /windows/system32/netapi32.dll /windows/system32/netapi32.dll
 
 COPY dist /

--- a/build/windows2016/nanoserver/Dockerfile
+++ b/build/windows2016/nanoserver/Dockerfile
@@ -1,4 +1,4 @@
-FROM mcr.microsoft.com/windows/servercore:2004 as core
+FROM mcr.microsoft.com/windows/servercore:ltsc2019 as core
 FROM microsoft/nanoserver:sac2016
 
 COPY --from=core /windows/system32/netapi32.dll /windows/system32/netapi32.dll

--- a/build/windows2016/nanoserver/Dockerfile
+++ b/build/windows2016/nanoserver/Dockerfile
@@ -1,5 +1,5 @@
 FROM mcr.microsoft.com/windows/servercore:ltsc2019 as core
-FROM microsoft/nanoserver:sac2016
+FROM mcr.microsoft.com/windows/nanoserver:1809-amd64
 
 COPY --from=core /windows/system32/netapi32.dll /windows/system32/netapi32.dll
 

--- a/build/windows2016/nanoserver/Dockerfile
+++ b/build/windows2016/nanoserver/Dockerfile
@@ -1,4 +1,4 @@
-FROM mcr.microsoft.com/windows/servercore as core
+FROM mcr.microsoft.com/windows/servercore:2004 as core
 FROM microsoft/nanoserver:sac2016
 
 COPY --from=core /windows/system32/netapi32.dll /windows/system32/netapi32.dll

--- a/build/windows2016/nanoserver/Dockerfile
+++ b/build/windows2016/nanoserver/Dockerfile
@@ -1,7 +1,7 @@
 FROM mcr.microsoft.com/windows/servercore as core
 FROM microsoft/nanoserver:sac2016
 
-COPY --from=core /windows/system32/netapi32.dll
+COPY --from=core /windows/system32/netapi32.dll /windows/system32/netapi32.dll
 
 COPY dist /
 

